test/functional/adb-commands-e2e-specs.js

```diff
@@ -48,7 +48,7 @@ describe('adb commands', function () {
   });
   it('defaultIME should get default IME', async function () {
     const defaultIME = await adb.defaultIME();
-    if (defaultIME) {
+    if (defaultIME && defaultIME !== 'null') {
```

Comment:
we should probably mention that `defaultIME` can return `null` strings in the docstring